drizzlepac/devutils/comparison_tools/integrate_read_hla_catalog.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+import os
+import pdb
+import sys
+from astropy.table import Table
+from drizzlepac.devutils.comparison_tools import compare_sourcelists
+from drizzlepac.devutils.comparison_tools.read_hla import read_hla_catalog
+
```

Comment:
I am not sure what this routine is doing in this directory.  This code looks like it is fully integrated in hapsequencer.  If this is true, this file should be deleted.

Reply:
Yeah. this was just a test script I wrote to test the correct_hla_classic_ra_dec() subroutine without having to take the time to run a full runhapsequencer.py run. I'll delete it.

Reply:
Deleted in commit [e6e8d8e](https://github.com/spacetelescope/drizzlepac/pull/548/commits/e6e8d8e30f08eca61e3a2212e09708b5deca43e6). Issue resolved.